database/rrd.h

```diff
@@ -458,8 +459,8 @@ struct rrdset {
 
     char *plugin_name;                              // the name of the plugin that generated this
     char *module_name;                              // the name of the plugin module that generated this
-
-    size_t unused[5];
+    uuid_t *chart_uuid;                             // Store the global GUID for this chart
```

Comment:
It is too late to change this, but be aware that `size_t` is not guaranteed to be the pointer size across all architectures (and that `uintptr_t` is more appropriate for this use).

Reply:
Well that's been in netdata since forever and it's part of the on-disk signatures for `memory mode = save/map` :(

Reply:
Btw Judy has a very nice type `Word_t`.